src/main/kotlin/no/nav/familie/ef/sak/behandling/dto/RevurderingDto.kt

```diff
@@ -8,5 +9,5 @@ data class RevurderingDto(
     val fagsakId: UUID,
     val behandlingsårsak: BehandlingÅrsak,
     val kravMottatt: LocalDate,
-    val barn: List<RevurderingBarnDto> = emptyList()
+    val vilkårsbehandleNyeBarn: VilkårsbehandleNyeBarn
```

Comment:
Vi lister opp disse barna i frontend i dag - er det ikke ønskelig lenger? 🤔 Funksjonelt spørsmål du kanskje bør ta med Lars...

Reply:
Denne brukes kun for requests inn i ef-sak, de som listes opp er fra 
```kotlin
@GetMapping("fagsak/{fagsakId}")
    fun nyeBarnSidenForrigeBehandling(
        @PathVariable("fagsakId")
        fagsakId: UUID
    ): Ressurs<BehandlingBarnDto> {
```

Reply:
ahaaa :)